doc/run/repl.md

```diff
@@ -1,16 +1,18 @@
-# Run SQLFlow REPL
+# Run SQLFlow command-line tool
```

Comment:
Rename the file to `cli.md` or `sqlflow_cli.md` or sth. like?

Reply:
done, we use cli.md